pkg/storage/backend/swift/config.go

```diff
@@ -0,0 +1,46 @@
+package swift
+
+import (
+	"flag"
+)
+
+// Config holds the config options for GCS backend
+type Config struct {
+	AuthUrl           string `yaml:"auth_url"`
+	Username          string `yaml:"username"`
+	UserDomainName    string `yaml:"user_domain_name"`
+	UserDomainID      string `yaml:"user_domain_id"`
+	UserId            string `yaml:"user_id"`
+	Password          string `yaml:"password"`
+	DomainId          string `yaml:"domain_id"`
+	DomainName        string `yaml:"domain_name"`
+	ProjectID         string `yaml:"project_id"`
+	ProjectName       string `yaml:"project_name"`
+	ProjectDomainID   string `yaml:"project_domain_id"`
+	ProjectDomainName string `yaml:"project_domain_name"`
+	RegionName        string `yaml:"region_name"`
+	ContainerName     string `yaml:"container_name"`
+}
+
+// RegisterFlags registers the flags for TSDB Swift storage
+func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
+	cfg.RegisterFlagsWithPrefix("blocks-storage.", f)
+}
+
+// RegisterFlagsWithPrefix registers the flags for TSDB Swift storage with the provided prefix
+func (cfg *Config) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
```

Comment:
Few things, please:
1. Capitalise `OpenStack` in CLI flags description. Also I would refer to it as `OpenStack Swift` and not just `OpenStack`
2. Make sure each CLI flag description ends with a dot `.`
3. `id` > `ID`

Reply:
Done